src/katex.less

```diff
@@ -475,6 +477,8 @@
     .stretchy {
         width: 100%;
         display: block;
+        position: relative;
+        overflow: hidden;
```

Comment:
Is this also b/c of a Chrome rendering issue?

Reply:
No. Those two new lines are only necessary for IE. In earlier work, I had put those two lines of CSS into class `hide-tail` and then nested `hide-tail` inside a `stretchy` span. I found here that I could save one nesting level by writing those two new lines into `stretchy`.  I could possibly refactor some other code and save a nesting level elsewhere, but I've got other priorities right now.